packages/components/src/Link/Link.stories.tsx

```diff
@@ -151,22 +151,20 @@ mediumVariantsOnDarkBackground.parameters = {
 export const linkWithIcon = Template.bind(null);
 linkWithIcon.args = {
   ...defaultArgs,
-  children: (
-    <>
-      Link <CheckCircleRoundedIcon purpose="decorative" />
-    </>
-  ),
+  children: "Link",
+  icon: {
+    iconElement: <CheckCircleRoundedIcon purpose="decorative" />,
```

Comment:
side note -- I wonder if it'd make sense for icons to default to `purpose: 'decorative'`

Reply:
I think the idea was to force people to always choose, and think about which purpose they're selecting.

Reply:
Oh, I'll update the story that has just an icon so it follows that pattern, in case anyone uses it as a guide.